src/components/projects/projectId/attributes/attributeId/ExecutionContainer.tsx

```diff
@@ -113,10 +126,7 @@ export default function ExecutionContainer(props: ExecutionContainerProps) {
                                         <div className="flex items-center justify-center mr-5 ml-auto">
                                             <KernButton
                                                 text="View"
-                                                onClick={() => {
-                                                    dispatch(setModalStates(ModalEnum.VIEW_RECORD_DETAILS, { open: true, recordIdx: index }));
-                                                    recordByRecordId(sampleRecords.recordIds[index]);
-                                                }}
+                                                onClick={viewRecordDetails(index)}
```

Comment:
doesn't work as reference for an onClick (will set it to void/undefined) onclick usually can't be set with a function call.